packages/sdk/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "tsdx test",
+    "test:watch": "tsdx test --watch",
+    "eslint": "eslint './src/**/*.ts'",
+    "prettier": "prettier './**/*.md' './src/**/*.ts'",
+    "lint": "yarn prettier --write && yarn eslint --fix",
```

Comment:
this was actually added when i was hacking around to get linting working. the standard tsdx is `tsdx lint` which i could not get working.

Reply:
because this package does not contain an eslint config or prettier config, i think it ends up using the repos. I'm going to leave this in for now, because husky does not seem to be working for me, and i have to run this in console.

Reply:
Have you tried reinstalling the packages?

Go to the root of the repo and run `yarn`.